hack/test

```diff
@@ -95,9 +96,10 @@ function build_restore_operator  {
 }
 
 function build_pass {
-	IMAGE=$OPERATOR_IMAGE hack/build/operator/build
-	build_backup_operator
-	build_restore_operator
+	# cache all builds by passing '-i'. 
+	IMAGE=$OPERATOR_IMAGE hack/build/operator/build -i
```

Comment:
Don't enable install in CI.

Reply:
should we enable cache in CI so that is will be faster to run? I thought that's the purpose of the support caching for go build.

Reply:
>  I thought that's the purpose of the support caching for go build.

No. We add it for local dev purpose.